src/ray/gcs/tables.h

```diff
@@ -513,7 +513,8 @@ class ProfileTable : private Log<UniqueID, ProfileTableData> {
   std::string DebugString() const;
 };
 
-using CustomSerializerTable = Table<ClassID, CustomSerializerData>;
+// TODO(suquark): Is the change from "ClassID" to "ActorClassID" reasonable here?
```

Comment:
Actually no it is not, `CustomSerializerTable` is currently unused I believe, but it is actually intended to keep track of custom serializers for arbitrary Python classes, not just actor classes. Since we got rid of `ClassID`, we can probably just use `UniqueID` here.

Reply:
using UniqueID now